accounts/erd1qqqqqqqqqqqqqpgqaqxztq0y764dnet95jwtse5u5zkg92sfacts6h9su3.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "SALSA",
+  "description": "Staking Agency's Liquid Staking Algorithm",
+  "tags": [
+    "staking",
+    "liquid",
+    "liquidstaking"
+  ]
```

Comment:
missing `icon` field. check this for reference: https://github.com/multiversx/mx-assets/blob/master/accounts/erd1v4ms58e22zjcp08suzqgm9ajmumwxcy4hfkdc23gvynnegjdflmsj6gmaq.json